### DISCUSSION TITLE
Question about not using some options for CURL multi

### DISCUSSION THREAD

**Discussion by BuddhaProgram** [curl/curl#12074] [Q&A] [answered]
Hello, I have some questions regarding setting options for multi interface.

I have crafted my project around multi interface through this example: https://curl.se/libcurl/c/ephiperfifo.html

I would like to know if it is possible not to use CURLOPT_LOW_SPEED_TIME and CURLOPT_LOW_SPEED_LIMIT for multi? Say for example, I want to have a really hard limit for both connect timeout (CURLOPT_CONNECTTIMEOUT) and timeout (CURLOPT_TIMEOUT) instead of relying also on low speed. Would it be possible to do that?

**Comment by bagder** [comment, marked as answer]
Yes (of course)
